map_server/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ find_package(catkin REQUIRED
 find_package(Bullet REQUIRED)
 find_package(SDL REQUIRED)
 find_package(SDL_image REQUIRED)
+find_package(OpenCV REQUIRED COMPONENTS core imgcodecs)
```

Comment:
is there some way to do this without bringing OpenCV -- that's a pretty heavy dependency for something as simple as the map saver?

Reply:
Since `SDL_image` is unable to write images. `libpng` may be ok, but a little troublesome. So I choose the `OpenCV` library. OpenCV is included by ROS, so I think it is not a big problem. But I will listen to your suggestion.